### PR TITLE
[ISSUE ##3832] in order to improve the performance by change the variable location of queueId

### PR DESCRIPTION
**Pull request by Aaron-TangCode**
**Make sure set the target branch to `develop`**

## What is the purpose of the change
[#3832](https://github.com/apache/rocketmq/issues/3832)
## Brief changelog

image

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Review by RongtongJin — APPROVED** [review]

**Review by dongeforever — APPROVED** [review]

**Review by Kvicii — CHANGES_REQUESTED** [review]
The wildcard import is not allowed. Revert.

**Comment by XiaoyiPeng** [comment]
> The wildcard import is not allowed. Revert.

+1

File: store/src/main/java/org/apache/rocketmq/store/CommitLog.java [hunk 1]
```diff
@@ -16,30 +16,10 @@
...
-
 import org.apache.rocketmq.common.ServiceThread;
 import org.apache.rocketmq.common.UtilAll;
 import org.apache.rocketmq.common.constant.LoggerName;
-import org.apache.rocketmq.common.message.MessageAccessor;
-import org.apache.rocketmq.common.message.MessageConst;
-import org.apache.rocketmq.common.message.MessageDecoder;
-import org.apache.rocketmq.common.message.MessageExt;
-import org.apache.rocketmq.common.message.MessageExtBatch;
+import org.apache.rocketmq.common.message.*;
```
Comment (XiaoyiPeng) [inline store/src/main/java/org/apache/rocketmq/store/CommitLog.java:22]: It is not recommended to import all classes with wildcard.

File: store/src/main/java/org/apache/rocketmq/store/CommitLog.java [hunk 2]
```diff
@@ -50,6 +30,13 @@
 import org.apache.rocketmq.store.ha.HAService;
 import org.apache.rocketmq.store.schedule.ScheduleMessageService;
 
+import java.net.*;
```
Comment (XiaoyiPeng) [inline store/src/main/java/org/apache/rocketmq/store/CommitLog.java:33]: It is not recommended to import all classes with wildcard.

File: store/src/main/java/org/apache/rocketmq/store/CommitLog.java (hunk 1 above)
Comment (caigy) [reply store/src/main/java/org/apache/rocketmq/store/CommitLog.java:22]: +1

File: store/src/main/java/org/apache/rocketmq/store/CommitLog.java (hunk 2 above)
Comment (caigy) [reply store/src/main/java/org/apache/rocketmq/store/CommitLog.java:33]: +1

**Review by caigy — CHANGES_REQUESTED** [review]

**Comment by Aaron-TangCode** [comment]
the problem about wildcard have fixed

**Review by duhenglucky — APPROVED** [review]

**Pull request merged**